tests/e2e/volumepropverification_test.go

```diff
@@ -69,15 +69,15 @@ Test steps:
 func TestVolumeProperties(t *testing.T) {
 	for vmIndx := 0; vmIndx < len(vms); vmIndx++ {
 		log.Println("running end_to_end tests - current time: ", time.Now())
-		vmIp = vms[vmIndx]
-		log.Println("Running test on VM - ", vmIp)
-		dockerCliCheck = verifier.IsDockerCliCheckNeeded(vms[vmIndx])
+		vmIP = vms[vmIndx]
+		log.Println("Running test on VM - ", vmIP)
+		dockerCliCheck = verification.IsDockerCliCheckNeeded(vms[vmIndx])
 		for i := 0; i < len(volSizes); i++ {
 			for k := 0; k < len(formatTypes); k++ {
 				containerName = "busybox_" + strconv.FormatInt(time.Now().Unix(), 20)
```

Comment:
there is already a task to address this https://github.com/vmware/docker-volume-vsphere/issues/1096